tests/test_functional.py

```diff
@@ -252,6 +253,74 @@ def test_Raster(self):
             o.rast.ST_Value(bottom_right_point)).scalar()
         assert bottom_right is None
 
+    def test_AddPoint(self):
```

Comment:
I don't think we need need to test AddPoint in the functional test. Please remove the test entirely.